point-of-sale-app/ui/src/components/Main.vue

```diff
@@ -0,0 +1,109 @@
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<template>
+  <div class="container">
+    <div class="row" id="pos">
+      <div class="col-md-8 col-md-offset-2">
+        <h1>Point of Sale</h1>
+        <div class="row">
+          <div class="col-md-6">
+            <transaction
+              :items="currentItems"
+              @remove="removeItem"
+              @edit="toggleEdit"
+            ></transaction>
+          </div>
+          <div class="col-md-6">
+            <item-list :items="items" @add="addItem"></item-list>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import ItemList from "./ItemList.vue";
+import Transaction from "./Transaction.vue";
+
+export default {
+  name: "Main",
+  title: 'Retail Edge POS',
+  components: {
+    ItemList,
+    Transaction,
+  },
+  props: {},
+  data: () => {
+    return {
+      title: 'Foo Page',
+      items: [
+        { id: 1, name: "Apple", price: 2.99 },
+        { id: 2, name: "Orange", price: 0.99 },
+        { id: 3, name: "Banana", price: 5.99 },
```

Comment:
LOL, the 6 dollar banana.

Question: BTW, I haven't run this sample app on my own machine, so this might be a stupid question... but is this JavaScript list of products currently being used? I thought the list of products and prices would be loaded from a back-end service (e.g., the inventory service).

Reply:
That is correct. This PR only introduces the UI elements and the next PR that you already reviewed is where I have added the code to consume the APIs and load them from the API